<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calender_window</class>
 <widget class="QWidget" name="calender_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>select a day by click on a day in the calender below and then click on show  to see it's task(tasks) information!</string>
   </property>
  </widget>
  <widget class="QTableView" name="taskslist_tableview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>851</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="calendertable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>851</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="show_pushButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>260</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>SHOW the tasks on this date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="months_comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="year_number_spinBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="showcalender_pushbutton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show calender!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>360</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to tasks window </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
